### ISSUE TITLE
Unable to start sunshine

### ISSUE THREAD

**Issue by amol-** [LizardByte/Sunshine#1049]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the nightly release?

- [X] This issue is present in the nightly release

### Describe the Bug

Sunshine doesn't start on Windows 10. It prints info about the various resolution and codecs, and then just quits. I was able to start it only once and never again.

### Expected Behavior

Sunshine starts

### Host Operating System

Windows

### Operating System Version

10

### Architecture

64 bit

### Sunshine commit or version

https://github.com/LizardByte/Sunshine/commit/e20ca7a8f023ca330f1dd409a5f13aef48b0daf2

### Package

Windows - portable

### GPU Type

AMD

### GPU Model

Radeon R9 290x

### GPU Driver/Mesa Version

27.20.20913.2000

### Config

```Shell
key_rightalt_to_key_win = disabled
gamepad = x360
upnp = disabled
dwmflush = enabled
min_log_level = 1
origin_pin_allowed = pc
origin_web_ui_allowed = lan
hevc_mode = 0
nv_preset = p4
nv_tune = ull
nv_coder = auto
nv_rc = cbr
qsv_preset = medium
qsv_coder = auto
amd_coder = auto
amd_preanalysis = disabled
amd_quality = balanced
amd_rc = vbr_latency
amd_usage = ultralowlatency
amd_vbaq = enabled
vt_coder = auto
vt_software = auto
vt_realtime = enabled
```

### Relevant log output

```shell
c:\Sunshine>sunshine.exe
[vt_coder] -- [auto]
[amd_vbaq] -- [enabled]
[amd_rc] -- [vbr_latency]
[amd_preanalysis] -- [disabled]
[vt_realtime] -- [enabled]
[key_rightalt_to_key_win] -- [disabled]
[amd_usage] -- [ultralowlatency]
[origin_pin_allowed] -- [pc]
[qsv_coder] -- [auto]
[gamepad] -- [x360]
[amd_coder] -- [auto]
[nv_preset] -- [p4]
[upnp] -- [disabled]
[nv_coder] -- [auto]
[dwmflush] -- [enabled]
[min_log_level] -- [1]
[vt_software] -- [auto]
[amd_quality] -- [balanced]
[origin_web_ui_allowed] -- [lan]
[nv_rc] -- [cbr]
[hevc_mode] -- [0]
[nv_tune] -- [ull]
[qsv_preset] -- [medium]
[2023:03:19:18:45:23]: Info: Sunshine version: 0.18.4.e20ca7a8f023ca330f1dd409a5f13aef48b0daf2
[2023:03:19:18:45:23]: Info: Compiling shaders...
[2023:03:19:18:45:23]: Info: System tray created
[2023:03:19:18:45:23]: Debug: System tray loop
[2023:03:19:18:45:23]: Info: Compiled shaders
[2023:03:19:18:45:23]: Info: // Testing for available encoders, this may generate errors. You can safely ignore those errors. //
[2023:03:19:18:45:23]: Info: Trying encoder [nvenc]
[2023:03:19:18:45:23]: Info: ddprobe.exe [1] [] returned: 0x00000000
[2023:03:19:18:45:23]: Info: Set GPU preference: 1
[2023:03:19:18:45:23]: Info:
Device Description : AMD Radeon R9 200 Series
Device Vendor ID   : 0x00001002
Device Device ID   : 0x00006810
Device Video Mem   : 2027 MiB
Device Sys Mem     : 0 MiB
Share Sys Mem      : 4070 MiB
Feature Level      : 0x0000B100
Capture size       : 1024x768
Offset             : 0x0
Virtual Desktop    : 1024x768
[2023:03:19:18:45:23]: Info: Desktop resolution [1024x768]
[2023:03:19:18:45:23]: Info: Desktop format [DXGI_FORMAT_B8G8R8A8_UNORM]
[2023:03:19:18:45:23]: Info:
Colorspace         : DXGI_COLOR_SPACE_RGB_FULL_G22_NONE_P709
Bits Per Color     : 8
Red Primary        : [0,0]
Green Primary      : [0,0]
Blue Primary       : [0,0]
White Point        : [0,0]
Min Luminance      : 0 nits
Max Luminance      : 0 nits
Max Full Luminance : 0 nits
[2023:03:19:18:45:23]: Info: SDR color coding [Rec. 601]
[2023:03:19:18:45:23]: Info: Color range: [JPEG]
[2023:03:19:18:45:23]: Error: Could not open codec [h264_nvenc]: Operation not permitted
[2023:03:19:18:45:23]: Info:
Device Description : AMD Radeon R9 200 Series
Device Vendor ID   : 0x00001002
Device Device ID   : 0x00006810
Device Video Mem   : 2027 MiB
Device Sys Mem     : 0 MiB
Share Sys Mem      : 4070 MiB
Feature Level      : 0x0000B100
Capture size       : 1024x768
Offset             : 0x0
Virtual Desktop    : 1024x768
[2023:03:19:18:45:23]: Info: Desktop resolution [1024x768]
[2023:03:19:18:45:23]: Info: Desktop format [DXGI_FORMAT_B8G8R8A8_UNORM]
[2023:03:19:18:45:23]: Info:
Colorspace         : DXGI_COLOR_SPACE_RGB_FULL_G22_NONE_P709
Bits Per Color     : 8
Red Primary        : [0,0]
Green Primary      : [0,0]
Blue Primary       : [0,0]
White Point        : [0,0]
Min Luminance      : 0 nits
Max Luminance      : 0 nits
Max Full Luminance : 0 nits
[2023:03:19:18:45:23]: Info: SDR color coding [Rec. 601]
[2023:03:19:18:45:23]: Info: Color range: [JPEG]
[2023:03:19:18:45:23]: Error: Could not open codec [h264_nvenc]: Operation not permitted
[2023:03:19:18:45:23]: Info:
Device Description : AMD Radeon R9 200 Series
Device Vendor ID   : 0x00001002
Device Device ID   : 0x00006810
Device Video Mem   : 2027 MiB
Device Sys Mem     : 0 MiB
Share Sys Mem      : 4070 MiB
Feature Level      : 0x0000B100
Capture size       : 1024x768
Offset             : 0x0
Virtual Desktop    : 1024x768
[2023:03:19:18:45:23]: Info: Desktop resolution [1024x768]
[2023:03:19:18:45:23]: Info: Desktop format [DXGI_FORMAT_B8G8R8A8_UNORM]
[2023:03:19:18:45:23]: Info:
Colorspace         : DXGI_COLOR_SPACE_RGB_FULL_G22_NONE_P709
Bits Per Color     : 8
Red Primary        : [0,0]
Green Primary      : [0,0]
Blue Primary       : [0,0]
White Point        : [0,0]
Min Luminance      : 0 nits
Max Luminance      : 0 nits
Max Full Luminance : 0 nits
[2023:03:19:18:45:23]: Info: SDR color coding [Rec. 601]
[2023:03:19:18:45:23]: Info: Color range: [JPEG]
[2023:03:19:18:45:23]: Error: Could not open codec [h264_nvenc]: Operation not permitted
[2023:03:19:18:45:24]: Info:
Device Description : AMD Radeon R9 200 Series
Device Vendor ID   : 0x00001002
Device Device ID   : 0x00006810
Device Video Mem   : 2027 MiB
Device Sys Mem     : 0 MiB
Share Sys Mem      : 4070 MiB
Feature Level      : 0x0000B100
Capture size       : 1024x768
Offset             : 0x0
Virtual Desktop    : 1024x768
[2023:03:19:18:45:24]: Info: Desktop resolution [1024x768]
[2023:03:19:18:45:24]: Info: Desktop format [DXGI_FORMAT_B8G8R8A8_UNORM]
[2023:03:19:18:45:24]: Info:
Colorspace         : DXGI_COLOR_SPACE_RGB_FULL_G22_NONE_P709
Bits Per Color     : 8
Red Primary        : [0,0]
Green Primary      : [0,0]
Blue Primary       : [0,0]
White Point        : [0,0]
Min Luminance      : 0 nits
Max Luminance      : 0 nits
Max Full Luminance : 0 nits
[2023:03:19:18:45:24]: Info: SDR color coding [Rec. 601]
[2023:03:19:18:45:24]: Info: Color range: [JPEG]
[2023:03:19:18:45:24]: Error: Could not open codec [h264_nvenc]: Operation not permitted
[2023:03:19:18:45:24]: Info: Encoder [nvenc] failed
[2023:03:19:18:45:24]: Info: Trying encoder [quicksync]
[2023:03:19:18:45:24]: Info:
Device Description : AMD Radeon R9 200 Series
Device Vendor ID   : 0x00001002
Device Device ID   : 0x00006810
Device Video Mem   : 2027 MiB
Device Sys Mem     : 0 MiB
Share Sys Mem      : 4070 MiB
Feature Level      : 0x0000B100
Capture size       : 1024x768
Offset             : 0x0
Virtual Desktop    : 1024x768
[2023:03:19:18:45:24]: Info: Desktop resolution [1024x768]
[2023:03:19:18:45:24]: Info: Desktop format [DXGI_FORMAT_B8G8R8A8_UNORM]
[2023:03:19:18:45:24]: Info:
Colorspace         : DXGI_COLOR_SPACE_RGB_FULL_G22_NONE_P709
Bits Per Color     : 8
Red Primary        : [0,0]
Green Primary      : [0,0]
Blue Primary       : [0,0]
White Point        : [0,0]
Min Luminance      : 0 nits
Max Luminance      : 0 nits
Max Full Luminance : 0 nits
[2023:03:19:18:45:24]: Info: SDR color coding [Rec. 601]
[2023:03:19:18:45:24]: Info: Color range: [JPEG]
[2023:03:19:18:45:24]: Error: Failed to derive device context: Unknown error occurred
[2023:03:19:18:45:24]: Info:
Device Description : AMD Radeon R9 200 Series
Device Vendor ID   : 0x00001002
Device Device ID   : 0x00006810
Device Video Mem   : 2027 MiB
Device Sys Mem     : 0 MiB
Share Sys Mem      : 4070 MiB
Feature Level      : 0x0000B100
Capture size       : 1024x768
Offset             : 0x0
Virtual Desktop    : 1024x768
[2023:03:19:18:45:24]: Info: Desktop resolution [1024x768]
[2023:03:19:18:45:24]: Info: Desktop format [DXGI_FORMAT_B8G8R8A8_UNORM]
[2023:03:19:18:45:24]: Info:
Colorspace         : DXGI_COLOR_SPACE_RGB_FULL_G22_NONE_P709
Bits Per Color     : 8
Red Primary        : [0,0]
Green Primary      : [0,0]
Blue Primary       : [0,0]
White Point        : [0,0]
Min Luminance      : 0 nits
Max Luminance      : 0 nits
Max Full Luminance : 0 nits
[2023:03:19:18:45:24]: Info: SDR color coding [Rec. 601]
[2023:03:19:18:45:24]: Info: Color range: [JPEG]
[2023:03:19:18:45:24]: Error: Failed to derive device context: Unknown error occurred
[2023:03:19:18:45:24]: Info:
Device Description : AMD Radeon R9 200 Series
Device Vendor ID   : 0x00001002
Device Device ID   : 0x00006810
Device Video Mem   : 2027 MiB
Device Sys Mem     : 0 MiB
Share Sys Mem      : 4070 MiB
Feature Level      : 0x0000B100
Capture size       : 1024x768
Offset             : 0x0
Virtual Desktop    : 1024x768
[2023:03:19:18:45:24]: Info: Desktop resolution [1024x768]
[2023:03:19:18:45:24]: Info: Desktop format [DXGI_FORMAT_B8G8R8A8_UNORM]
[2023:03:19:18:45:24]: Info:
Colorspace         : DXGI_COLOR_SPACE_RGB_FULL_G22_NONE_P709
Bits Per Color     : 8
Red Primary        : [0,0]
Green Primary      : [0,0]
Blue Primary       : [0,0]
White Point        : [0,0]
Min Luminance      : 0 nits
Max Luminance      : 0 nits
Max Full Luminance : 0 nits
[2023:03:19:18:45:24]: Info: SDR color coding [Rec. 601]
[2023:03:19:18:45:24]: Info: Color range: [JPEG]
[2023:03:19:18:45:24]: Error: Failed to derive device context: Unknown error occurred
[2023:03:19:18:45:24]: Info:
Device Description : AMD Radeon R9 200 Series
Device Vendor ID   : 0x00001002
Device Device ID   : 0x00006810
Device Video Mem   : 2027 MiB
Device Sys Mem     : 0 MiB
Share Sys Mem      : 4070 MiB
Feature Level      : 0x0000B100
Capture size       : 1024x768
Offset             : 0x0
Virtual Desktop    : 1024x768
[2023:03:19:18:45:24]: Info: Desktop resolution [1024x768]
[2023:03:19:18:45:24]: Info: Desktop format [DXGI_FORMAT_B8G8R8A8_UNORM]
[2023:03:19:18:45:24]: Info:
Colorspace         : DXGI_COLOR_SPACE_RGB_FULL_G22_NONE_P709
Bits Per Color     : 8
Red Primary        : [0,0]
Green Primary      : [0,0]
Blue Primary       : [0,0]
White Point        : [0,0]
Min Luminance      : 0 nits
Max Luminance      : 0 nits
Max Full Luminance : 0 nits
[2023:03:19:18:45:24]: Info: SDR color coding [Rec. 601]
[2023:03:19:18:45:24]: Info: Color range: [JPEG]
[2023:03:19:18:45:24]: Error: Failed to derive device context: Unknown error occurred
[2023:03:19:18:45:24]: Info: Encoder [quicksync] failed
[2023:03:19:18:45:24]: Info: Trying encoder [amdvce]
[2023:03:19:18:45:24]: Info:
Device Description : AMD Radeon R9 200 Series
Device Vendor ID   : 0x00001002
Device Device ID   : 0x00006810
Device Video Mem   : 2027 MiB
Device Sys Mem     : 0 MiB
Share Sys Mem      : 4070 MiB
Feature Level      : 0x0000B100
Capture size       : 1024x768
Offset             : 0x0
Virtual Desktop    : 1024x768
[2023:03:19:18:45:24]: Info: Desktop resolution [1024x768]
[2023:03:19:18:45:24]: Info: Desktop format [DXGI_FORMAT_B8G8R8A8_UNORM]
[2023:03:19:18:45:24]: Info:
Colorspace         : DXGI_COLOR_SPACE_RGB_FULL_G22_NONE_P709
Bits Per Color     : 8
Red Primary        : [0,0]
Green Primary      : [0,0]
Blue Primary       : [0,0]
White Point        : [0,0]
Min Luminance      : 0 nits
Max Luminance      : 0 nits
Max Full Luminance : 0 nits
[2023:03:19:18:45:24]: Info: SDR color coding [Rec. 601]
[2023:03:19:18:45:24]: Info: Color range: [JPEG]
[2023:03:19:18:45:25]: Info:
Device Description : AMD Radeon R9 200 Series
Device Vendor ID   : 0x00001002
Device Device ID   : 0x00006810
Device Video Mem   : 2027 MiB
Device Sys Mem     : 0 MiB
Share Sys Mem      : 4070 MiB
Feature Level      : 0x0000B100
Capture size       : 1024x768
Offset             : 0x0
Virtual Desktop    : 1024x768
[2023:03:19:18:45:25]: Info: Desktop resolution [1024x768]
[2023:03:19:18:45:25]: Info: Desktop format [DXGI_FORMAT_B8G8R8A8_UNORM]
[2023:03:19:18:45:25]: Info:
Colorspace         : DXGI_COLOR_SPACE_RGB_FULL_G22_NONE_P709
Bits Per Color     : 8
Red Primary        : [0,0]
Green Primary      : [0,0]
Blue Primary       : [0,0]
White Point        : [0,0]
Min Luminance      : 0 nits
Max Luminance      : 0 nits
Max Full Luminance : 0 nits
[2023:03:19:18:45:25]: Info: SDR color coding [Rec. 601]
[2023:03:19:18:45:25]: Info: Color range: [JPEG]
[2023:03:19:18:45:25]: Info:
Device Description : AMD Radeon R9 200 Series
Device Vendor ID   : 0x00001002
Device Device ID   : 0x00006810
Device Video Mem   : 2027 MiB
Device Sys Mem     : 0 MiB
Share Sys Mem      : 4070 MiB
Feature Level      : 0x0000B100
Capture size       : 1024x768
Offset             : 0x0
Virtual Desktop    : 1024x768
[2023:03:19:18:45:25]: Info: Desktop resolution [1024x768]
[2023:03:19:18:45:25]: Info: Desktop format [DXGI_FORMAT_B8G8R8A8_UNORM]
[2023:03:19:18:45:25]: Info:
Colorspace         : DXGI_COLOR_SPACE_RGB_FULL_G22_NONE_P709
Bits Per Color     : 8
Red Primary        : [0,0]
Green Primary      : [0,0]
Blue Primary       : [0,0]
White Point        : [0,0]
Min Luminance      : 0 nits
Max Luminance      : 0 nits
Max Full Luminance : 0 nits
[2023:03:19:18:45:25]: Info: SDR color coding [Rec. 601]
[2023:03:19:18:45:25]: Info: Color range: [JPEG]
[2023:03:19:18:45:26]: Info:
Device Description : AMD Radeon R9 200 Series
Device Vendor ID   : 0x00001002
Device Device ID   : 0x00006810
Device Video Mem   : 2027 MiB
Device Sys Mem     : 0 MiB
Share Sys Mem      : 4070 MiB
Feature Level      : 0x0000B100
Capture size       : 1024x768
Offset             : 0x0
Virtual Desktop    : 1024x768
[2023:03:19:18:45:26]: Info: Desktop resolution [1024x768]
[2023:03:19:18:45:26]: Info: Desktop format [DXGI_FORMAT_B8G8R8A8_UNORM]
[2023:03:19:18:45:26]: Info:
Colorspace         : DXGI_COLOR_SPACE_RGB_FULL_G22_NONE_P709
Bits Per Color     : 8
Red Primary        : [0,0]
Green Primary      : [0,0]
Blue Primary       : [0,0]
White Point        : [0,0]
Min Luminance      : 0 nits
Max Luminance      : 0 nits
Max Full Luminance : 0 nits
[2023:03:19:18:45:26]: Info: SDR color coding [Rec. 601]
[2023:03:19:18:45:26]: Info: Color range: [JPEG]
[2023:03:19:18:45:26]: Error: Could not open codec [hevc_amf]: Encoder not found
[2023:03:19:18:45:26]: Info:
Device Description : AMD Radeon R9 200 Series
Device Vendor ID   : 0x00001002
Device Device ID   : 0x00006810
Device Video Mem   : 2027 MiB
Device Sys Mem     : 0 MiB
Share Sys Mem      : 4070 MiB
Feature Level      : 0x0000B100
Capture size       : 1024x768
Offset             : 0x0
Virtual Desktop    : 1024x768
[2023:03:19:18:45:26]: Info: Desktop resolution [1024x768]
[2023:03:19:18:45:26]: Info: Desktop format [DXGI_FORMAT_B8G8R8A8_UNORM]
[2023:03:19:18:45:26]: Info:
Colorspace         : DXGI_COLOR_SPACE_RGB_FULL_G22_NONE_P709
Bits Per Color     : 8
Red Primary        : [0,0]
Green Primary      : [0,0]
Blue Primary       : [0,0]
White Point        : [0,0]
Min Luminance      : 0 nits
Max Luminance      : 0 nits
Max Full Luminance : 0 nits
[2023:03:19:18:45:26]: Info: SDR color coding [Rec. 601]
[2023:03:19:18:45:26]: Info: Color range: [JPEG]
[2023:03:19:18:45:26]: Error: Could not open codec [hevc_amf]: Encoder not found
[2023:03:19:18:45:26]: Info:
Device Description : AMD Radeon R9 200 Series
Device Vendor ID   : 0x00001002
Device Device ID   : 0x00006810
Device Video Mem   : 2027 MiB
Device Sys Mem     : 0 MiB
Share Sys Mem      : 4070 MiB
Feature Level      : 0x0000B100
Capture size       : 1024x768
Offset             : 0x0
Virtual Desktop    : 1024x768
[2023:03:19:18:45:26]: Info: Desktop resolution [1024x768]
[2023:03:19:18:45:26]: Info: Desktop format [DXGI_FORMAT_B8G8R8A8_UNORM]
[2023:03:19:18:45:26]: Info:
Colorspace         : DXGI_COLOR_SPACE_RGB_FULL_G22_NONE_P709
Bits Per Color     : 8
Red Primary        : [0,0]
Green Primary      : [0,0]
Blue Primary       : [0,0]
White Point        : [0,0]
Min Luminance      : 0 nits
Max Luminance      : 0 nits
Max Full Luminance : 0 nits
[2023:03:19:18:45:26]: Info: SDR color coding [Rec. 601]
[2023:03:19:18:45:26]: Info: Color range: [JPEG]
[2023:03:19:18:45:26]: Error: Could not open codec [hevc_amf]: Encoder not found
[2023:03:19:18:45:26]: Info:
Device Description : AMD Radeon R9 200 Series
Device Vendor ID   : 0x00001002
Device Device ID   : 0x00006810
Device Video Mem   : 2027 MiB
Device Sys Mem     : 0 MiB
Share Sys Mem      : 4070 MiB
Feature Level      : 0x0000B100
Capture size       : 1024x768
Offset             : 0x0
Virtual Desktop    : 1024x768
[2023:03:19:18:45:26]: Info: Desktop resolution [1024x768]
[2023:03:19:18:45:26]: Info: Desktop format [DXGI_FORMAT_B8G8R8A8_UNORM]
[2023:03:19:18:45:26]: Info:
Colorspace         : DXGI_COLOR_SPACE_RGB_FULL_G22_NONE_P709
Bits Per Color     : 8
Red Primary        : [0,0]
Green Primary      : [0,0]
Blue Primary       : [0,0]
White Point        : [0,0]
Min Luminance      : 0 nits
Max Luminance      : 0 nits
Max Full Luminance : 0 nits
[2023:03:19:18:45:26]: Info: SDR color coding [Rec. 601]
[2023:03:19:18:45:26]: Info: Color range: [JPEG]
[2023:03:19:18:45:26]: Error: Could not open codec [hevc_amf]: Encoder not found
[2023:03:19:18:45:27]: Info:
Device Description : AMD Radeon R9 200 Series
Device Vendor ID   : 0x00001002
Device Device ID   : 0x00006810
Device Video Mem   : 2027 MiB
Device Sys Mem     : 0 MiB
Share Sys Mem      : 4070 MiB
Feature Level      : 0x0000B100
Capture size       : 1024x768
Offset             : 0x0
Virtual Desktop    : 1024x768
[2023:03:19:18:45:27]: Info: Desktop resolution [1024x768]
[2023:03:19:18:45:27]: Info: Desktop format [DXGI_FORMAT_B8G8R8A8_UNORM]
[2023:03:19:18:45:27]: Info:
Colorspace         : DXGI_COLOR_SPACE_RGB_FULL_G22_NONE_P709
Bits Per Color     : 8
Red Primary        : [0,0]
Green Primary      : [0,0]
Blue Primary       : [0,0]
White Point        : [0,0]
Min Luminance      : 0 nits
Max Luminance      : 0 nits
Max Full Luminance : 0 nits
[2023:03:19:18:45:27]: Info: SDR color coding [Rec. 709]
[2023:03:19:18:45:27]: Info: Color range: [JPEG]

c:\Sunshine>
```

**Comment by ReenigneArcher** [comment]
Duplicate of #945
